.travis.yml

```diff
@@ -89,35 +90,48 @@ install:
   ;;
   esac
 - cargo install --list
+
 script: |
   case "$FEATURE" in
 
   "fmt" )
       cd exonum && cargo update && cd .. &&
       cd sandbox && cargo update && cd .. &&
+      cd testkit && cargo update && cd .. &&
       cd exonum && cargo audit && cd .. &&
+      cd testkit && cargo audit && cd .. &&
       cd exonum && cargo fmt -- --write-mode=diff && cd .. &&
-      cd sandbox && cargo fmt -- --write-mode=diff && cd ..
+      cd sandbox && cargo fmt -- --write-mode=diff && cd .. &&
+      cd testkit && cargo fmt -- --write-mode=diff && cd ..
   ;;
   "clippy-core" )
       cd exonum && cargo clippy --verbose --features long_benchmarks  -- -D warnings
   ;;
   "clippy-sandbox" )
       cd sandbox && cargo clippy --verbose -- -D warnings
   ;;
+  "clippy-testkit" )
+      cd testkit && cargo clippy --verbose -- -D warnings
+  ;;
   "test-core" )
       cargo test --manifest-path exonum/Cargo.toml --verbose --tests --lib --features "float_serialize"
   ;;
   "test-sandbox" )
-      cargo test --manifest-path sandbox/Cargo.toml  --verbose
+      cargo test --manifest-path sandbox/Cargo.toml --verbose
+  ;;
+  "test-testkit" )
+      cargo test --manifest-path testkit/Cargo.toml --verbose
```

Comment:
I think it's preferable to use `-p` rather than `--manifest-path` here.

Reply:
The same in theory applies to `cargo clippy` and `cargo deadlink`, but I am not sure that those subcommands handle `-p` correctly.